api/migrations/versions/0197_creating_service_template_when_a_.py

```diff
@@ -0,0 +1,59 @@
+"""
+
+Revision ID: 0197
+Revises: 0196
+Create Date: 2018-08-24 14:31:34.732360
+
+"""
+from alembic import op
+from datetime import datetime
+
+revision = '0197'
+down_revision = '0196'
+
+template_id = '831139ba-a984-11e8-814c-80e6501bb488'
+
+content = """Hi Notify Support,
+
+The following service has requested to go live: 
+
+Name: ((service_name)) 
+Url: ((service_url))
```

Comment:
@andrewoh531 and it's minor but while you are renaming this template can you use `URL` instead of `Url`.

Reply:
Sure